### ISSUE TITLE
Autofill-button for Twitter DANK faucet/[FEATURE]

### ISSUE THREAD

**Issue by cosmohm** [district0x/memefactory#754]
# Problem

Current process to claim free $DANK from $DANK faucet is not user-friendly. Should get automated with a button that auto-fills your tweet. The message should also be more marketing-oriented. _(See below)_

## Implementation

Add a button that auto-fills a message on the user's Twitter that says the following: 

"I'm verifying my address [_address_] and claiming my free $DANK tokens on @memefactory0x so I can vote and challenge memes. 

Go claim your free $DANK on #Polygon today: https://memefactory.io/get-dank."

<img width="976" alt="Screen Shot 2022-03-15 at 4 47 56 PM" src="https://user-images.githubusercontent.com/80524059/158485082-6c78a538-993b-43ca-a3f0-128fd977d109.png">

**Comment by madvas** [comment]
yes good idea, let's add button which links to twitter with the prefilled text. 
The button can be where "@MemeFactory0x $DANK 0x1a2b3c4d...5e6f" currently is.
The button text: "Make a tweet", preferabbly with the twitter icon. 
The Tweet URL input stays unchanged.
